src/traces/scattermapbox/hover.js

```diff
@@ -34,6 +42,7 @@ module.exports = function hoverPoints(pointData, xval, yval) {
     function distFn(d) {
         var lonlat = d.lonlat;
         if(lonlat[0] === BADNUM) return Infinity;
+        if(trace.cluster.enabled && clusteredPointsIds.indexOf(d.i + 1) === -1) return Infinity;
```

Comment:
The hover does not appear!
But if you revert the changes if `hover.js` it displays the original data points; which I think is desirable.

Reply:
The hover functionality should at the moment work when the individual points is unpacked from the cluster. 

I think it is desirable to enable the hover functionality is being enabled when the individual points is unpacked from its cluster